dsgrid/utils/dataset.py

```diff
@@ -139,6 +141,21 @@ def add_column_from_records(df, dimension_records, dimension_name, column_to_add
     return df
 
 
+def apply_scaling_factor(
+    df: DataFrame, value_columns, scaling_factor_column=SCALING_FACTOR_COLUMN
+) -> DataFrame:
+    """Apply the scaling factor to all value columns and then drop the scaling factor column."""
+    for column in value_columns:
+        df = df.withColumn(
+            column,
+            F.when(
+                F.col(scaling_factor_column).isNotNull(),
+                F.col(column) * F.col(scaling_factor_column),
+            ).otherwise(F.col(column)),
+        )
```

Comment:
When does the scaling factor column from load_data_lookup get joined to the load_data table? I don't see that happening in dataset_schema_handler_standard.py.

Reply:
Lines 63 and 102 of dataset_schema_handler_standard.py